packages/store/src/state-stream.ts

```diff
@@ -1,20 +1,12 @@
-import { Injectable, Optional, SkipSelf } from '@angular/core';
+import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs/BehaviorSubject';
 
 /**
  * BehaviorSubject of the entire state.
  */
 @Injectable()
 export class StateStream extends BehaviorSubject<any> {
-  constructor(
-    @Optional()
-    @SkipSelf()
-    parent: StateStream
-  ) {
+  constructor() {
```

Comment:
If we remove all that, do we event need the constructor now?

Reply:
I think something breaks if there is no immediate value